src/utils/LocaleUtil.ts

```diff
@@ -3,9 +3,11 @@ import { IDatePicker } from '../common/@types';
 import { range } from '../utils/ArrayUtil';
 import * as localeData from 'dayjs/plugin/localeData';
 import * as localizedFormat from 'dayjs/plugin/localizedFormat';
+import * as weekday from 'dayjs/plugin/weekday';
 
 dayjs.extend(localeData);
 dayjs.extend(localizedFormat);
+dayjs.extend(weekday)
```

Comment:
```suggestion
dayjs.extend(weekday);
```
it's lint error, missing semicolon

Reply:
oh thank you very much. I'll fix it asap!